homeassistant/components/bmw_connected_drive/sensor.py

```diff
@@ -519,68 +192,8 @@ def __init__(
         else:
             self._attr_native_unit_of_measurement = description.unit_metric
```

Comment:
We can clean up some parameters and attributes now from the entity init. `service` parameter and `self._service` attribute aren't used anymore.

Reply:
They might be coming back in one of the future versions of the library. If needed I can remove them though.

Reply:
Yes, please remove code we don't use for now.